packages/editor/src/managers/clipboard/copy-cut-manager.ts

```diff
@@ -216,6 +214,8 @@ export class CopyCutManager {
     } finally {
       tempElem.removeEventListener('copy', listener);
       document.body.removeChild(tempElem);
+
+      const curRange = getCurrentRange();
```

Comment:
Please revert this change, it will break copy action in the format bar (selection lost)


https://user-images.githubusercontent.com/18554747/218300412-d4dea242-c77d-4433-97ed-b3a6f93a50b8.mov